src/Form/DataTransformer/TagArrayToStringTransformer.php

```diff
@@ -36,7 +36,7 @@ public function __construct(ObjectManager $manager)
     /**
      * {@inheritdoc}
      */
-    public function transform($array)
+    public function transform($array): string
```

Comment:
`array` typehint is missing. What if we rename the argument too from `$array` to `$tags` ?

Reply:
> `array` typehint is missing.

Yep, but I can't do that, because declaration must be compatible with `DataTransformerInterface->transform(value)`.

> What if we rename the argument too from $array to $tags ?

:+1: Done.